### PR TITLE
swiper.el (swiper--action): Set evil search direction to forward

### PR DESCRIPTION
**Pull request by edkolev**
Without setting evil-ex-search-direction, evil-search-next and
evil-search-previous (bound to `n` and `N` in evil-mode) will repeat
swiper's last search, but will use the direction of the pre-swiper
search. If there is no previous search before swiper, evil-search-next
and evil-search-previous will not work. swiper doesn't have a concept
of `backward` search, hence it makes sense to always set direction to
'forward.

### CODE REVIEW

**Comment by edkolev** [comment]
The commit is a bit wordy, please let me know if the RP can be improved.

This will fix https://github.com/emacs-evil/evil/issues/712

**Comment by abo-abo** [comment]
Thanks.

> The commit message is a bit wordy

That's how I like them:)

**Comment by edkolev** [comment]
Nice, thanks again :)
